data-types.md

```diff
@@ -122,8 +122,10 @@ A standard point of vehicle telemetry. References to latitude and longitude impl
 | `journey_id`      | UUID            | Required               | If telemetry occurred during a trip, the ID of the journey.  If not in a trip, `null`.
 | `stop_id`         | UUID            | Required if Applicable | Stop that the vehicle is currently located at. Only applicable for _docked_ Micromobility. See [Stops][stops] |
 | `location`        | [GPS][gps]      | Required               | Telemetry position data |
+| `location_type`   | Enum            | Required if Known      | If detectable and known, what type of location the device is on or in. One of `street`, `sidewalk`, `crosswalk`, `garage`, `bike_lane`.   |
```

Comment:
For `location_type ` does this look right to you @droverai? See #767

Reply:
Yes, that looks good!